orc8r/gateway/python/magma/common/service.py

```diff
@@ -90,6 +90,9 @@ def __init__(self, name, empty_mconfig, loop=None):
         self._start_time = int(time.time())
         self._register_signal_handlers()
 
+        if workers is None:
+            workers = 10
```

Comment:
minor nit: should this be made a config variable instead

Reply:
Agree, maybe for another PR :)